public/app/core/components/Breadcrumb/Breadcrumb.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import { useSelector } from 'react-redux';
+import { dataQa } from '@percona/platform-core';
+import { act } from 'react-dom/test-utils';
+import { Breadcrumb } from './Breadcrumb';
+
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useSelector: jest.fn(),
```

Comment:
Is this mock required if you are mocking it in beforeEach anyway?

Reply:
Yes, because in `beforeEach` I'm mocking the implementation, it wouldn't work without `jest.mock`